extensions/sql-migration/src/dialog/selectMigrationService/selectMigrationServiceDialog.ts

```diff
@@ -338,7 +338,7 @@ export class SelectMigrationServiceDialog {
 	private async _populateAzureAccountsDropdown(): Promise<void> {
 		try {
 			this._azureAccountsDropdown.loading = true;
-			this._azureAccounts = await getAzureAccounts();
+			this._azureAccounts = await getAzureAccounts(TelemetryViews.SelectMigrationServiceDialog);
```

Comment:
I suggest not passing in telemetry names, in getAzureAccounts function just report the error source as "Utility.getAzureAccounts"

Reply:
Agreed, that's what I had originally but figured the context would've been helpful. I can revert this to clean it up a bit